### PR TITLE
remove duplicate properties keys

### LINKED ISSUE #9169: Duplicate property key in Bundle.properties file V5.12.1
Hi Dataverse Community,

We detected a duplicate keys in the Bundle.properties file : 

advanced.search.datasets.persistentId.tip=The persistent identifier for the Dataset.
advanced.search.datasets.persistentId.tip=The Dataset's unique persistent identifier, either a DOI or Handle

best regards
Steven Ferey.

### PR DESCRIPTION
**Pull request by stevenferey**
**What this PR does / why we need it**:

Remove duplicate property keys from property files

**Which issue(s) this PR closes**:

Closes #9169

**Special notes for your reviewer**:

**Suggestions on how to test this**:

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

**Is there a release notes update needed for this change?**:

No

**Additional documentation**:

### CODE REVIEW

**Comment by pdurbin** [comment]
@stevenferey thanks for the pull request!

I have no idea why but edu.harvard.iq.dataverse.api.FilesIT.testAccessFacet failed:

Screen Shot 2023-07-19 at 4 45 44 PM

From https://jenkins.dataverse.org/job/IQSS-Dataverse-Develop-PR/job/PR-9718/1/testReport/junit/edu.harvard.iq.dataverse.api/FilesIT/testAccessFacet/

**Comment by qqmyers** [comment]
FWIW: I've seen this on another PR - #9599. Not sure what it is yet.

**Review by qqmyers — APPROVED** [review]
Yep, all duplicates - thanks for the cleanup!

File: src/main/java/propertyFiles/Bundle.properties [hunk 1]
```diff
@@ -961,7 +960,6 @@ advanced.search.files.variableName=Variable Name
 advanced.search.files.variableName.tip=The name of the variable's column in the data frame.
 advanced.search.files.variableLabel=Variable Label
 advanced.search.files.variableLabel.tip=A short description of the variable.
-advanced.search.datasets.persistentId.tip=The persistent identifier for the Dataset.
 advanced.search.datasets.persistentId=Persistent Identifier
 advanced.search.datasets.persistentId.tip=The Dataset's unique persistent identifier, either a DOI or Handle
```
Comment (qqmyers) [inline src/main/java/propertyFiles/Bundle.properties:964]: Same idea - Given that we now allow PermaLinks and people can add new PID types, perhaps the shorter version is better? i.e. keep line 964 and remove this one?

Comment (jggautier) [reply src/main/java/propertyFiles/Bundle.properties:964]: I think this would resolve https://github.com/IQSS/dataverse/issues/9740, too

**Review by qqmyers — CHANGES_REQUESTED** [review]
@stevenferey - Can you flip which lines are kept as requested in the comments? Assuming that's OK, we can move this forward once you're done.

**Comment by qqmyers** [comment]
@stevenferey - can you make the changes requested above so this can move forward. (If not, let us know if you're OK with the change and we can get someone else to edit.)

**Review by qqmyers — APPROVED** [review]
I've gone ahead and changed to use the shorter versions of two duplicates (ones that don't mention DOIs and Handles which should also now include PermaLinks, etc.). With that - the other changes look good and this should be able to move forward.

**Pull request merged**